src/conversions.jl

```diff
@@ -47,20 +47,20 @@ convert(::Type{RGB24},   x::Real) = RGB24(x)
 convert(::Type{ARGB32},  x::Real) = ARGB32(x)
 convert(::Type{Gray24},  x::Real) = Gray24(x)
 convert(::Type{AGray32}, x::Real) = AGray32(x)
-convert(::Type{ARGB32},  x::Real, alpha) = ARGB32(x, alpha)
```

Comment:
I don't understand how this should have worked, so I removed it (since I was unable to find a way to test it). Please correct me if I misunderstood (In that case I could use a few hints what to do here)